lib/streamlit/config.py

```diff
@@ -1052,89 +1052,116 @@ def _browser_server_port() -> int:
     categories=["theme"],
     description="""
         The preset Streamlit theme that your custom theme inherits from.
-        One of "light" or "dark".
+        This can be one of the following: "light" or "dark".
     """,
 )
 
 _create_theme_options(
     "primaryColor",
     categories=["theme", CustomThemeCategories.SIDEBAR],
-    description="Primary accent color for interactive elements.",
+    description="""
+        Primary accent color for interactive elements.
+    """,
 )
 
 _create_theme_options(
     "backgroundColor",
     categories=["theme", CustomThemeCategories.SIDEBAR],
-    description="Background color for the main content area.",
+    description="""
+        Background color of the app.
+    """,
 )
 
 _create_theme_options(
     "secondaryBackgroundColor",
     categories=["theme", CustomThemeCategories.SIDEBAR],
-    description="Background color used for the sidebar and most interactive widgets.",
+    description="""
+        Background color used for most interactive widgets.
+    """,
 )
 
 _create_theme_options(
     "textColor",
     categories=["theme", CustomThemeCategories.SIDEBAR],
-    description="Color used for almost all text.",
+    description="""
+        Color used for almost all text.
+    """,
 )
 
 _create_theme_options(
     "linkColor",
     categories=["theme", CustomThemeCategories.SIDEBAR],
-    description="Color used for all links.",
+    description="""
+        Color used for all links.
+    """,
 )
 
 _create_theme_options(
     "codeBackgroundColor",
     categories=["theme", CustomThemeCategories.SIDEBAR],
-    description="Background color used for code blocks.",
+    description="""
+        Background color used for code blocks.
+    """,
 )
 
 _create_theme_options(
     "font",
     categories=["theme", CustomThemeCategories.SIDEBAR],
     description="""
-        The font family for all text in the app, except code blocks. One of "sans serif",
-        "serif", or "monospace".
-        To use a custom font, it needs to be added via [theme.fontFaces].
+        The font family for all text, except code blocks. This can be one of
+        the following: "sans serif", "serif", "monospace", or the
```

Comment:
I think its better to suggest `sans-serif` instead of `sans serif` to actually use the valid font family value. We do support `sans serif` for legacy, but it shouldn't be the preferred choice.